api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java

```diff
@@ -171,7 +171,7 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
             new UserAuthentication(user, OAuth2Userinfo, token, UserType.RESEARCHER));
 
     // TODO: setup this in the context, get rid of log statement
```

Comment:
```suggestion
    // TODO: Remove log statement and attach userinfo as a Stackdriver key/value instead.
    // This log line is currently the the only reliable way to associate a particular App Engine request log
    // with the authenticated user identity, which is critical information for debugging.
```

Reply:
@calbach if you can sketch out which properties you want in a Stackdriver log (and if you want it in JSON, which I recommend for filtering and making logs-based metrics), what keys & values would be helpful, I'll be happy to do the logging part.

Reply:
merging this in the meantime

Reply:
Just email is fine to start to keep parity here. I don't have other ideas off the top of my head (authority and data access level are also interesting)